cpp/include/table.hpp

```diff
@@ -128,6 +128,23 @@ struct table {
   gdf_size_type _num_rows{0};         ///< The number of elements in each column
 };
 
+/**---------------------------------------------------------------------------*
+ * @brief Returns vector of the dtypes of the columns in a table
+ *
+ * @param table The table to get the column dtypes from
+ * @return std::vector<gdf_dtype>
+ *---------------------------------------------------------------------------**/
+std::vector<gdf_dtype> column_dtypes(cudf::table const& table);
+
+/**---------------------------------------------------------------------------*
+ * @brief Indicates if a table contains any null values.
+ *
+ * @param table The table to check for null values
+ * @return true If the table contains one or more null values
+ * @return false If the table contains zero null values
+ *---------------------------------------------------------------------------**/
+bool have_nulls(cudf::table const& table);
```

Comment:
Since there is only one table passed as an argument, I think "have" should be "has".

```suggestion
bool has_nulls(cudf::table const& table);
```

Reply:
I went back and forth several times. My initial thinking was that the columns _have_ nulls. But I suppose it's more appropriate to say that the table _has_ nulls.